integration/spark2/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala

```diff
@@ -42,7 +42,8 @@ trait ServerInterface {
   /**
    * Used to prune and cache the datamaps for the table.
    */
-  def getSplits(request: DistributableDataMapFormat): Array[ExtendedBlocklet]
+  def getSplits(request: DistributableDataMapFormat, taskGroupId: String, taskGroupDesc: String):
```

Comment:
Its better to add task group id and taskGroupDesc in a wrapper otherise the query would be displayed in the logs which may cause security breach